fpdf/line_break.py

```diff
@@ -393,6 +393,13 @@ def add_character(
         active_fragment = self.fragments[-1]
 
         if character == SPACE:
+            # If we choose to break at space and we only have an empty fragment
+            # We will add a space to print, so that width doesn't evaluate to 0 when printing
+            if active_fragment.characters == []:
+                # add a space
```

Comment:
Please don't do that. Fix the problem where it is happening, and don't change the data being processed.